play/plaintext-descriptions/en-US.txt

```diff
@@ -33,16 +33,14 @@ AndBible is an open-source community project. In practice, this means that anyon
 
 If you are a professional software engineer or tester, please consider contributing to the project. For more information on how to contribute, please see https://git.io/JUnaj.
 
-Support by buying development time!
+If you dont' ahve time or skills for code contribution, you can support AndBible by sponsoring development time.
```

Comment:
"don't have" 😄

Reply:
Need to make these changes to transifex (we use en_GB variant there for minor/grammar fixes to english).